app/src/forms/admin/routes.js

```diff
@@ -18,22 +18,175 @@ routes.use(currentUser);
 //
 // Forms
 //
+
+/**
+ * @openapi
+ * /admin/forms:
+ *  get:
```

Comment:
The old docs covered the `active` parameter, and it needs to be added here.

Reply:
Okay